ArduPlane/sensors.cpp

```diff
@@ -34,6 +34,9 @@ void Plane::read_rangefinder(void)
     rangefinder_height_update();
 }
 
+#if HAL_AIRSPEED_ENABLED
+// FIXME 
+// calc_airspeed_errors() needs to move
 /*
   ask airspeed sensor for a new value
  */
```

Comment:
So I've got the defines compiling to disable Airspeed on plane. I'm not even sure if that is a good idea...

But to complete that I need to move the following items out to another loop any ideas?  If they can get moved out it is one more function that could get moved up once I figure out how to handle the MASK_LOG_IMU define.

https://github.com/ArduPilot/ardupilot/blob/9578b9bcd431e9e29b2625fddb0115f00e081c3b/ArduPlane/sensors.cpp#L44-L53